cypress/e2e/checkout/checkout.notes.cy.js

```diff
@@ -27,50 +32,99 @@ describe('Check out - Notes', () => {
       Permissions.uiUsersView.gui,
     ]).then((createdUserProperties) => {
       testData = createdUserProperties;
-      servicePoint = NewServicePoint.getDefaultServicePoint();
-      ServicePoints.createViaApi(servicePoint);
+      ServicePoints.createViaApi(instanceData.servicePoint);
+      instanceData.defaultLocation = Location.getDefaultLocation(instanceData.servicePoint.id);
+      Location.createViaApi(instanceData.defaultLocation).then((location) => {
+        InventoryInstances.createFolioInstancesViaApi({
+          folioInstances: instanceData.folioInstances,
+          location,
+        });
+      });
       UserEdit.addServicePointViaApi(
-        servicePoint.id,
+        instanceData.servicePoint.id,
         createdUserProperties.userId,
-        servicePoint.id,
+        instanceData.servicePoint.id,
       );
       cy.login(testData.username, testData.password);
-      cy.visit(TopMenu.usersPath);
-      UsersSearchPane.waitLoading();
     });
   });
 
+  beforeEach('Create notes', () => {
+    cy.visit(TopMenu.usersPath);
+    UsersSearchPane.waitLoading();
+    UsersSearchPane.searchByUsername(testData.username);
+    UsersSearchPane.waitLoading();
+    // Scroll down to "Notes" accordion button and click on it.
+    usersCard.openNotesSection();
+    // Create Note 1
+    AgreementsDetails.createNote({ ...note1, checkoutApp: true });
+    // Scroll down to "Notes" accordion button and click on it.
+    usersCard.openNotesSection();
+    // Create Note 2
+    AgreementsDetails.createNote({ ...note2, checkoutApp: true });
+    // Navigate to "Check out" app
+    cy.visit(TopMenu.checkOutPath);
+  });
+
   after('Deleting created entities', () => {
-    UserEdit.changeServicePointPreferenceViaApi(testData.userId, [servicePoint.id]);
-    ServicePoints.deleteViaApi(servicePoint.id);
+    UserEdit.changeServicePointPreferenceViaApi(testData.userId, [instanceData.servicePoint.id]);
+    ServicePoints.deleteViaApi(instanceData.servicePoint.id);
+    InventoryInstances.deleteInstanceViaApi({
+      instance: instanceData.folioInstances[0],
+      servicePoint: instanceData.servicePoint,
+      shouldCheckIn: true,
+    });
     Users.deleteViaApi(testData.userId);
   });
 
   it(
-    'C356781: Verify that all notes assigned to user pop up when user scan patron card (“Delete” option) (Spitfire)',
-    { tags: [TestTypes.criticalPath, DevTeams.spitfire] },
+    'C356781: Verify that all notes assigned to user pop up when user scan patron card (“Delete” option) (Spitfire) (TaaS)',
+    { tags: [TestTypes.criticalPath, DevTeams.spitfire, Parallelization.nonParallel] },
```

Comment:
Do we need "Parallelization.nonParallel" here, could you please remove it if not critical to use.